cms/static/js/collections/asset.js

```diff
@@ -37,8 +37,7 @@ define([
             return PagingCollection.prototype.parse.call(this, response, options);
         },
 
-        /* jshint unused:false */
-        parseState: function (response, queryParams, state, options) {
+        parseState: function(response, queryParams, state, options) {  // eslint-disable-line no-unused-vars
```

Comment:
I'm anxious about modifying code I don't understand and I don't know the test coverage of in such a major way (ie, not just a spacing fix). I can try to figure out what the coverage status of this method is.